code/game/gamemodes/events/biomass.dm

```diff
@@ -27,13 +27,11 @@
 	if (!W || !user || !W.type)
 		return
 
-	switch(W.type)
+	switch(W.type) //This is absolutely terrible

```

Comment:
Just didn't want to change any kudzu killability balancing or w/e to keep my changes atomic, maybe fix the snowflake in another PR since even if it's puting ol' snowrlake out of it's misery it's still a balance change.

I'll do it when I get back home tho